community/history.md

```diff
@@ -0,0 +1,61 @@
+---
+title: History
+---
+
+# History of conda-forge
+
+conda-forge's origins cannot be explained without understanding the context of Python packaging back in the early 2010s. Back then, the installation of Python packages across operating systems was very challenging, specially on Windows, as it often meant compiling dependencies from source.
+
+Python 2.x was the norm, the community was transitioning from `easy_install` to `pip`, and there wouldn't be an alternative for Python eggs [^eggs] until 2012, when wheels are introduced [^wheels]. To get Python, you'd get the official installers from Python.org, stick to the system provided one in Linux, or resort to ActiveState's or Enthought's distributions in macOS and Windows [^legacy-python-downloads].
```

Comment:
Python(x,y) was a popular Windows distribution back then, especially for science users. I recall it and Enthought's basically being the two main options for getting things working on Windows.
```suggestion
Python 2.x was the norm, the community was transitioning from `easy_install` to `pip`, and there wouldn't be an alternative for Python eggs [^eggs] until 2012, when wheels are introduced [^wheels]. To get Python, you'd get the official installers from Python.org, stick to the system provided one in Linux, or resort to options like ActiveState's or Enthought's distributions in macOS and Windows [^legacy-python-downloads].
```

Reply:
Added "options like" to not disregard Python(x,y) and others.